### ISSUE TITLE
Beaming after a triplet bracket doesn't seem to render a single beam

### ISSUE THREAD

**Issue by boblete** [opensheetmusicdisplay/opensheetmusicdisplay#1071]
![image](https://user-images.githubusercontent.com/445490/137879474-6a425a66-b242-4615-8305-739504d1381d.png)
The manual bracket works, but the single beam is not honoured.

**Comment by pnome** [comment]
I would like to see this too.  If you have bracketing turned off for the triplet, then it's particularly tricky to read.
I'm attaching a test file which I believe should render with a single beam between the group of 16ths and the group of 16th triplets but it doesn't.

To clarify, this is what it looks like:

![Image](https://github.com/user-attachments/assets/6378c6a9-f777-448a-975c-875b3d654c9b)

And this is how I believe it should look (this one has a bracket, but I can't see a way to only enable brackets on full triplets if they're beamed to something else)

![Image](https://github.com/user-attachments/assets/9f65da0f-d5f4-4560-8288-e09b50839775)

[beaming 16 and triplet.txt](https://github.com/user-attachments/files/19758674/beaming.16.and.triplet.txt)

**Comment by sschmidTU** [comment]
Fixed.

<img width="423" height="139" alt="Image" src="https://github.com/user-attachments/assets/1c2631e7-f3d3-4144-9776-55b162ffad16" />

Added as test sample `test_beam_secondary_beam_break_16th_and_triplet.musicxml`.

@boblete Your sample actually didn't have the right beam breaks in the MusicXML. I've added them, then OSMD does the breaks:

<img width="836" height="181" alt="Image" src="https://github.com/user-attachments/assets/f3eedb7c-dde8-4d78-8370-4c8b3669bd8c" />

[Triplet Beam Test_beams_corrected.txt](https://github.com/user-attachments/files/25376390/Triplet.Beam.Test_beams_corrected.txt)

no negative visual regressions, just secondary beam breaks now executed in 3 (other) existing test samples:
Haydn Concertante
TelemannWV40.102_Sonate-Nr.1.1-Dolce
test_drum_tuplet_flat_beams_Siebener_Endstreich_testFlam

[visual_diff_secondary_beam_breaks.zip](https://github.com/user-attachments/files/25376405/visual_diff_secondary_beam_breaks.zip)

**Comment by pnome** [comment]
Thanks for looking into it. Will this be in 1.9.7?

**Comment by sschmidTU** [comment]
Yes, this will be in the next release version, which should be 1.9.7. Maybe in a few days, maybe tomorrow or so, let's see how many unreleased changes there are / will be.

**Comment by sschmidTU** [comment]
@pnome This is now released in OSMD 1.9.7.
